pkg/otlp/metrics/metrics_translator.go

```diff
@@ -202,9 +202,17 @@ func (t *Translator) mapNumberMonotonicMetrics(
 			continue
 		}
 
-		if dx, ok := t.prevPts.MonotonicDiff(pointDims, startTs, ts, val); ok {
+		dx, firstPoint, dropPoint := t.prevPts.MonotonicDiff(pointDims, startTs, ts, val)
```

Comment:
nit: I think being a bit more verbose here helps readability
```suggestion
		dx, isFirstPoint, shouldDropPoint := t.prevPts.MonotonicDiff(pointDims, startTs, ts, val)
```